### PR TITLE
Trev8939/blend renderer

### PR DESCRIPTION
**Pull request by TADraeseke**
Next release of the SDK will no longer allow creating a new `RasterLayer`s from a `Raster` which is already used to create an existing `RasterLayer`.  Fix creates new `RasterLayer`s when updating the renderer.

### CODE REVIEW

File: blend-renderer/src/main/java/com/esri/arcgisruntime/sample/blendrenderer/MainActivity.java [hunk 1]
```diff
@@ -29,6 +29,10 @@
 
   private MapView mMapView;
 
+  private File mImageFile;
+
+  private File mElevationFile;
+
   private Raster mImageryRaster;
 
   private Raster mElevationRaster;
```
Comment (doneill) [inline blend-renderer/src/main/java/com/esri/arcgisruntime/sample/blendrenderer/MainActivity.java]: The `Raster` fields above should be converted to local variables.

Comment (doneill) [inline blend-renderer/src/main/java/com/esri/arcgisruntime/sample/blendrenderer/MainActivity.java]: Reduce the amount of spaces between your globals.

**Comment by TADraeseke** [comment]
@doneill global -> local variables done.

As to the whitespace, Shelly's said I ought to start using the [auto-format file](https://devtopia.esri.com/runtime/java-common-api/wiki/Java-and-Android-code-guidelines) (which introduces that extra whitespace between global variables). If there's a different format on samples, happy to change it!

**Comment by doneill** [comment]
@TADraeseke The guideline doc you reference needs to be updated, although I don't see where it states every variable needs a line space between them.  Otherwise updates look good to merge.

**Comment by TADraeseke** [comment]
@doneill When Code > Reformat Code is run using the ESRI_RTC_Java scheme, it adds all those line spaces. I can remove them manually or merge as is, depending on your preference.

**Comment by doneill** [comment]
@TADraeseke Hmm, ok, yeah we will need to update that as it may be ok for SDK code we can pass more globals around in app facing code.  Block them up for now and I will look to update our public facing format. 

```java
  private MapView mMapView;
  private Raster mImageryRaster;
  private Raster mElevationRaster;
  private SlopeType mSlopeType;

  private int mAltitude;
  private int mAzimuth;
  private double mZFactor;
  
  private ColorRamp.PresetType mColorRampType;
  private double mPixelSizeFactor;
  private double mPixelSizePower;
  private int mOutputBitDepth;
  private FragmentManager mFragmentManager;
```

**Comment by TADraeseke** [comment]
@doneill Reformatted, including blocking globals. Reformatting code also moved listener in ParametersDialogFragment to the bottom of the code.

**Comment by doneill** [comment]
@TADraeseke good to merge.

**Pull request merged**